# Generated by Django 3.0.4 on 2020-05-05 17:56

import django.contrib.auth.validators
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ("accounts", "0002_auto_20200327_1817"),
    ]

    operations = [
        migrations.AlterModelOptions(name="user", options={"ordering": ["-id"]},),
        migrations.AlterField(
            model_name="user",
            name="username",
            field=models.CharField(
                error_messages={"unique": "A user with that username already exists."},
                help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
                max_length=150,
                unique=True,
                validators=[
                    django.contrib.auth.validators.UnicodeUsernameValidator(),
                    django.core.validators.MinLengthValidator(3),
                ],
                verbose_name="username",
            ),
        ),
    ]
